tools/DevDiagnostics/DevHome.DevDiagnostics/Controls/InsightForMissingFileProcessTerminationControl.xaml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.DevDiagnostics.Controls.InsightForMissingFileProcessTerminationControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:DevHome.DevDiagnostics.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*"/>
+            <ColumnDefinition Width="Auto"/>
+        </Grid.ColumnDefinitions>
+
+        <TextBox Text="{x:Bind Text, Mode=OneWay}" AutomationProperties.AutomationId="ContentTextBock" BorderThickness="0" VerticalAlignment="Center" IsReadOnly="True" Background="Transparent"/>
+        <local:ElevatedButtonControl Text="{x:Bind ElevationButtonTextString}" Grid.Column="1" Margin="20 0 0 0" Command="{x:Bind Command, Mode=OneWay}" AutomationProperties.Name="{x:Bind ElevationButtonTextString}" >
```

Comment:
Isn't this a spot where you can use the 'preferred' x:UID.PropertyName automatic binding?

Reply:
Not in this case. I've made this a common control, and so any caller can create one of these and define it's text and tooltip. Maybe there is a magic way to AutomationProperties.Name bind to this control's text field instead of doing it the way I am, but I don't know if that buys much.